pkgs/development/python-modules/walrus/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, redis
+}:
+
+buildPythonPackage rec {
+  pname = "walrus";
+  version = "0.9.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "coleifer";
+    repo = "walrus";
+    rev = version;
+    hash = "sha256-jinYMGSBAY8HTg92qU/iU5vGIrrDr5SeQG0XjsBVfcc=";
+  };
+
+  propagatedBuildInputs = [
+    redis
+  ];
+
+  # requires redis instance running
+  doCheck = false;
```

Comment:
We may be able to run tests like `django-redis`.
https://github.com/NixOS/nixpkgs/blob/6500b4580c2a1f3d0f980d32d285739d8e156d92/pkgs/development/python-modules/django-redis/default.nix#L55C8-L62

Reply:
Fantastic, I love it!